CHANGES.md

```diff
@@ -13,6 +13,13 @@ ones in. -->
 
 ## 2.1.0 (<span actions:bind='release-date'>Released 2023-07-21</span>)
 
+### Breaking Changes
+
+[2736](https://github.com/metomi/rose/pull/2736)
```

Comment:
Under the wrong heading

Reply:
If you are relying on `PYTHONPATH` to add functions for rose ana this change will break your workflow. On one side of the change you must set `PYTHONPATH`, on the other `ROSE_PYTHONPATH`.